app/src/model/accounts.js

```diff
@@ -0,0 +1,66 @@
+const connection = require('./connection');
+
+class UserEntity {
+    /**
+     *
+     * @param {number} id
+     * @param {string} username
+     * @param {string} password
+     */
+
+    constructor(id, username, password) {
+        this.id = id;
+        this.username = username;
+        this.password = password;
+    }
+}
+/**
+ * @param {PostEntity} post
+ * @returns {Promise<undefind>}
+ * @throws {MysqlError}
+ */
+async function insertUser(accounts) {
```

Comment:
`accounts` to konta, przekazujesz tu wiele kont czy jedno?

Reply:
poprawione